config/locales/en.yml

```diff
@@ -2380,6 +2380,11 @@ See the %{link} to find out more about %{sitename}'s features and to start using
   shipping: "Shipping"
   shipping_methods: "Shipping Methods"
   payment_methods: "Payment Methods"
+  provider_settings: "Provider Settings"
+  active: "Active"
+  description: "Description"
+  tags: "Tags"
+  provider: "Provider"
```

Comment:
Sorry about this but I have handled this problem in a very recent PR:
https://github.com/openfoodfoundation/openfoodnetwork/pull/5425/files
I have actually moved these translations around so it's better if you delete these 5 keys from this PR. They will become obsolete after 5425.

Btw, if you want to learn something for next time, as we fix translations, we also try to make translations use lazy lookups, see here for details:
https://github.com/openfoodfoundation/openfoodnetwork/wiki/Internationalisation-(i18n)#development
One example is here:
https://github.com/openfoodfoundation/openfoodnetwork/pull/5425/files#diff-1985e8d0aa61624ade9de3f91944c7b3R19

Reply:
Thank you for the feedback @luisramos0. 

> Sorry about this but I have handled this problem in a very recent PR:
> https://github.com/openfoodfoundation/openfoodnetwork/pull/5425/files
> I have actually moved these translations around so it's better if you delete these 5 keys from this PR. They will become obsolete after 5425.
> 
> Btw, if you want to learn something for next time, as we fix translations, we also try to make translations use lazy lookups, see here for details:
> https://github.com/openfoodfoundation/openfoodnetwork/wiki/Internationalisation-(i18n)#development
> One example is here:
> https://github.com/openfoodfoundation/openfoodnetwork/pull/5425/files#diff-1985e8d0aa61624ade9de3f91944c7b3R19

I had been curious about how/when the lazy lookup is implemented so yes, always want to learn for next time :-)

Reply:
@luisramos0  - your comment mentioned removing 5 keys, but the code snippet shows 4. Can you please clarify which ones you want me to remove? Thanks!

Reply:
the snippet is 4 lines long, you have changed 5 lines, see in https://github.com/openfoodfoundation/openfoodnetwork/pull/5448/files

Reply:
@luisramos0. Deleted and will push again. Thanks